doc/source/f2py/code/pyproj_skbuild.toml

```diff
@@ -1,5 +1,3 @@
-[project]
-requires-python = ">=3.7"
-
 [build-system]
-requires = ["setuptools>=42", "wheel", "scikit-build", "cmake>=3.18", "numpy>=1.21"]
+requires = ["setuptools>=42", "wheel", "scikit-build", "cmake>=3.9", "numpy>=1.21"]
+build-backend = "setuptools.build_meta"
```

Comment:
Always specify the build backend. Otherwise this is legacy.

Reply:
(not related to this cleanup) The line above worries me, by the way. You probably want a recent NumPy so you can get a shiny new f2py with the latest fixes when making the output file, but you want an old version of NumPy for binary compatibly.

Reply:
Yeah there's no great way to deal with this. Maybe a comment explaining this trade-off should be added above this requirement? It can also link to https://numpy.org/devdocs/user/depending_on_numpy.html